site/src/modules/provisioners/useCompatibleProvisioners.ts

```diff
@@ -0,0 +1,24 @@
+import { ProvisionerDaemon } from "api/typesGenerated";
+
+export const provisionersUnhealthy = (provisioners : ProvisionerDaemon[]) => {
+	return provisioners.reduce((allUnhealthy, provisioner) => {
```

Comment:
I think what you really want here is [`every`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every), not `reduce`. `reduce` is almost never the best way to write something in JS. 😄

Reply:
this function is defunct and has been removed. thanks for the tip though. will remember it for future use.